### PR TITLE
fix null object reference

### PR DESCRIPTION
**Pull request by zidaneymar**
Fixes https://github.com/microsoft/BotFramework-Composer/issues/3729

## Description

Minor fix, activity from facebook channel take 'null' locale, this line would throw exception:
Object reference not set to an instance of an object.
Since the activity.Locale could be null.

### CODE REVIEW

**Comment by zidaneymar** [comment]
@gabog  Hi Gabo, Could you please help review this PR, should we always add default language policy in this recognizer? (Lu's comments)

**Comment by gabog** [comment]
> @gabog Gabo Gilabert FTE Hi Gabo, Could you please help review this PR, should we always add default language policy in this recognizer? (Lu's comments)

Sorry, I don't know enough about the design of this class and locale handling to comment on that.

File: libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Recognizers/MultiLanguageRecognizer.cs [hunk 1]
```diff
@@ -55,7 +55,7 @@ public override async Task<RecognizerResult> RecognizeAsync(DialogContext dialog
                 policy.AddRange(targetpolicy);
             }
 
-            if (activity.Locale.Length != 0 && LanguagePolicy.TryGetValue(string.Empty, out string[] defaultPolicy))
+            if (activity.Locale?.Length != 0 && LanguagePolicy.TryGetValue(string.Empty, out string[] defaultPolicy))
```
Comment (tomlm) [inline libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Recognizers/MultiLanguageRecognizer.cs:58]: >activity.Locale?.Length != 0 && [](start = 16, length = 31)

there should be no check against activity.Locale at all, remove this.

Comment (zidaneymar) [reply libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Recognizers/MultiLanguageRecognizer.cs]: resolved.

**Comment by zidaneymar** [comment]
Hi @tomlm , is this pr ready to merge

**Comment by luhan2017** [comment]
@tomlm , could you please help review and merge this PR?

**Review by EricDahlvang — APPROVED** [review]

**Review by tomlm — APPROVED** [review]
:shipit:

**Pull request merged**